ui/dashboards/src/context/DashboardProvider/DashboardProvider.tsx

```diff
@@ -133,17 +133,18 @@ function initStore(props: DashboardProviderProps) {
           ...createEditJsonDialogSlice(...args),
           metadata,
           display,
-          defaultTimeRange: { pastDuration: duration },
+          timeRange: { pastDuration: duration },
```

Comment:
Any reason for the renaming here? Guessing it's for clarity, but wanted to double-check I wasn't missing something else, especially since you left some of the hooks using "default time range" naming.

Reply:
Looking at it again, I actually think we should just rename this `timeRange` (prev defaultTimeRange) to `duration` since that is how it's defined in our dashboard schema and is what is stored. It also makes some of the logic in places like TimeRangeControls simpler. So now `useDefaultTimeRange` becomes `useDashboardDuration` and setDashboard calls `state.duration = duration;`

The full active timeRange from plugin-system (relative and absolute) is more relevant for runtime uses cases, whereas saving a full absolute time range doesn't make sense when storing a dashboard (confirmed this with Mohit too)